compiler/src/iree/compiler/Codegen/Common/GPU/test/BUILD.bazel

```diff
@@ -14,6 +15,12 @@ package(
     licenses = ["notice"],  # Apache 2.0
 )
 
+tests_requiring_rocm = [
```

Comment:
same comment here about testing this with the plugin if it's plugin-specific - treat plugins as their own subrepos